parsers/uottawa.js

```diff
@@ -37,7 +38,7 @@ function parse(text) {
       var professor = _professorRegex.exec(classes[j]);
       var time = _timeRegex.exec(classes[j]);
       var where = _locationRegex.exec(classes[j]);
-      if (!professor || !time || !where) continue;
```

Comment:
Some this means there are some calendar with no input of prof or location? Isn't there a default name